NAMESPACE

```diff
@@ -3,6 +3,102 @@
 S3method(plot,gtable)
 S3method(print,gtable)
 export("%~%")
+export(.CohortSizeConst)
```

Comment:
why are the exports of default constructors needed for linter fix?

Reply:
This is not related, I just had to run the `Document` `roxygen` so that the help pages are updated after `lintr` changes and then the `NAMESPACE` file was changed (as it is maintained by the `roxygen`).

Reply:
I do not know why it was not exported before, even that the `@export` tag was there before. We can maybe remove the `@export` tag from the default constructors.